hack/install-tackle.sh

```diff
@@ -100,12 +101,22 @@ kubectl wait \
   --namespace konveyor-tackle \
   --for=condition=Successful \
   --timeout=600s \
-  tackles.tackle.konveyor.io/tackle
+  tackles.tackle.konveyor.io/tackle \
+|| kubectl get \
+  --namespace konveyor-tackle \
+  -o yaml \
+  tackles.tackle.konveyor.io/tackle # Print tackle debug when timed out
 
 # Now wait for all the tackle deployments
 kubectl wait \
   --namespace konveyor-tackle \
   --selector="app.kubernetes.io/part-of=tackle" \
   --for=condition=Available \
   --timeout=600s \
-  deployments.apps
+  deployments.apps \
+|| kubectl get \
```

Comment:
Considered using kubectl describe, but that doesn't support filtering items with field-selector.